api/src/utils/redact.ts

```diff
@@ -86,3 +86,19 @@ export function redact(input: UnknownObject, paths: Paths, replacement: string):
 		}
 	}
 }
+
+/**
+ * Extract values from Error objects for use with JSON.stringify()
+ */
+export function errorReplacer(_key: string, value: any) {
```

Comment:
Can we use `unknown` here?
```suggestion
export function errorReplacer(_key: string, value: unknown) {
```

Reply:
Yes we should. Thanks!